bin/latex_to_unicode.py

```diff
@@ -177,6 +177,9 @@ def latex_to_unicode(s):
     # Transform errant citations into "(CITATION)"
     s = re.sub(r"\\(new)?cite.? ?\{([\w:-]+?)\}", r"(CITATION)", s)
 
+    # Remove errant \footnotes
+    s = re.sub(r"\\footnote ?\{.*?\}", "", s)
```

Comment:
Are you worried about cases like `\footnote{This is \emph{really} important.}`?

Reply:
Clearly not :) But I independently ran into this while reprocessing EMNLP (#736), where an (obvious) common instance is `\footnote{\url{...}}`. 

We really need to add unit tests so we know we're handling these issues correctly.

Reply:
Oh yes, that's an extremely common case. The lazy solution would be to delete `\footnote` but turn its argument into normal text.

Reply:
The non-lazy solution would be to add `\footnote` to the list of LaTeX commands, so that it gets parsed correctly with its argument. If you're sure this is the right way to go, I can try to make the modification. Am I allowed to push to your branch?

Something similar should probably be done for \cite and friends, although there are so many variants of this command that it would clutter up the list of commands...

Reply:
Yes, please do.